tests/ro/creator/data/roc-02-emobon-observatory.yml

```
@@ -0,0 +1,71 @@
+spatialCoverage:
+  at: 51.44333331, 2.808331
+  in: mrgid:26567
+temporalCoverage:  # same for all obs?
+  start: 2021-07-01
+  end: !resolve {today}
+license: https://creativecommons.org/licenses/by/4.0/ # same for all obs?
+# keywords and themes derived from 'strategy'
+
+agents:
+  "https://orcid.org/0000-0002-8559-3508":  # Klaas Deneudt
+    roles: [ contact, maintainer]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)